### PR TITLE
setup.py extension so it can be build and released to pypi

### PR DESCRIPTION
**Pull request by PhilipMay**
As written in #93 I created and uploaded a PyPI package of bilm.
This is the setup.py change I made and it would be cool if you merge it.
Thanks
Philip

### CODE REVIEW

**Comment by PhilipMay** [comment]
Hi, what about this PR?
Since I transfered ownership in PyPI to @matt-gardner you can chane him as maintainer...
But it would be nice to accept this PR.
Thanks,
Philip

**Comment by matt-gardner** [comment]
I didn't receive any notification about ownership transfer - maybe you meant @matt-peters?

File: setup.py [hunk 1]
```diff
@@ -1,13 +1,48 @@
...
+        'Topic :: Scientific/Engineering :: Information Analysis',
+        'Topic :: Software Development',
+        'Topic :: Software Development :: Libraries',
+        'Topic :: Software Development :: Libraries :: Python Modules',
+        'Topic :: Text Processing',
+    ],
+    keywords='bilm elmo nlp embedding',
+    author='Matthew E. Peters, Mark Neumann, Mohit Iyyer, Matt Gardner, Christopher Clark, Kenton Lee, Luke Zettlemoyer',
+    author_email='allennlp-contact@allenai.org',
+    maintainer='Philip May',
```
Comment (matt-peters) [inline setup.py]: Can you change this line to "Matthew Peters" and remove the subsequent line with `maintainer_email`?  Then I'll merge.

**Comment by PhilipMay** [comment]
Have done so...

**Comment by matt-peters** [comment]
Is it possible to leave out the tensorflow dependency in setup.py and require the user to manually install their preferred version of tensorflow?  Many developers (myself included) have multiple systems they use, including laptops without GPUs for development and servers with GPUs for training.

**Comment by PhilipMay** [comment]
Sure - good idea. I made the change...

**Comment by PhilipMay** [comment]
I also increased the version to 0.1.post5 - will build a new pip package when you acceped the pr. ;-)

**Comment by PhilipMay** [comment]
Ok - I made a new package for PyPI: https://pypi.org/project/bilm/

**Pull request merged**